python/tk_desktop2/websockets/requests/toolkit_actions/execute_action.py

```diff
@@ -122,6 +122,13 @@ def __init__(self, connection, id, parameters):
             # for project pages, project_id is passed down.
             self._project_id = parameters["project_id"]
 
+    @property
+    def analytics_command_name(self):
+        """
+        The command name to pass to analytics.
+        """
+        return "execute_toolkit_action"
```

Comment:
note - we are not tracking the details of _which_ action was carried out - this is already handled elsewhere.